### PR TITLE
Fix: update important information section in homepage

### PR DESCRIPTION
**Pull request by teoh4770**
Issue: #334 
Branch: [fix/334/update-important-information-section-in-home-page](https://github.com/LaurierHawkHacks/Dashboard/compare/dev...fix/334/update-important-information-section-in-home-page?expand=1)

Why this pr is needed:
1. to actually update Hawkhacks important date and time to homepage for participants to see

What has changed:
1. the content: I have created a data file (src/data/importantDateTimes.ts) to easily modify the important date time
2. the styling: I have changed the original styling to make the message clearer.

Sample image:

Screenshot 2024-05-07 at 7 49 27 PM

### CODE REVIEW

**Review by AmirAgassi — APPROVED** [review]
Nice! LGTM

File: src/data/importantDateTimes.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const importantDateTimes = [
+    {
+        date: "May 17, 2024",
+        label: "First Day",
+        events: {
+            "Registration Time": "6:00 PM - 11:00 PM",
+            "Opening Ceremony": "7:45 PM - 9:20 PM",
+            "Hacking Begin": "9:20 PM",
```
Comment (GabeDiniz) [inline src/data/importantDateTimes.ts:8]: I think it should be "Hacking Begins"

File: src/data/importantDateTimes.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        },
+    },
+    {
+        date: "May 19, 2024",
+        label: "Last Day",
+        events: {
+            "Hacking End": "10:00 AM",
```
Comment (GabeDiniz) [inline src/data/importantDateTimes.ts:15]: And this would be "Hacking Ends"

**Review by GabeDiniz — APPROVED** [review]
Other than that, LGTM! :))

**Pull request merged**